payload-files/astro/_test-param-error_[param]--transaction.json

```diff
@@ -95,60 +107,73 @@
         "RequestData",
         "Console",
         "Http",
-        "Undici",
+        "NodeFetch",
         "OnUncaughtException",
         "OnUnhandledRejection",
         "ContextLines",
-        "LocalVariables",
+        "LocalVariablesAsync",
         "Context",
-        "Modules"
+        "Express",
+        "Fastify",
+        "Graphql",
+        "Mongo",
+        "Mongoose",
+        "Mysql",
+        "Mysql2",
+        "Redis",
+        "Postgres",
+        "Nest",
+        "Hapi",
+        "Koa",
+        "Connect"
       ],
       "name": "sentry.javascript.astro",
       "packages": [
         {
           "name": "npm:@sentry/astro",
-          "version": "7.113.0"
+          "version": "8.0.0-beta.6"
         },
         {
           "name": "npm:@sentry/node",
-          "version": "7.113.0"
+          "version": "8.0.0-beta.6"
         }
       ],
-      "version": "7.113.0"
+      "version": "8.0.0-beta.6"
     },
     "server_name": "D9M3PY4LQ7.local",
     "spans": [
       {
         "data": {
+          "http.client": "fetch",
           "http.method": "GET",
           "http.response.status_code": 500,
+          "http.status_code": 500,
+          "http.target": "/api/test-param-error/1337",
+          "http.url": "http://localhost:4321/api/test-param-error/1337",
```

Comment:
not parameterized

Reply:
this is fine, this is "only" outgoing fetch stuff!